scripts/verify/verify-py.sh

```diff
@@ -28,7 +28,7 @@ run_flake8() {
     fi
 
     local pyfiles
-    pyfiles=$(find . -type d \( -path ./_output -o -path ./vendor -o -path ./assets -o -path ./etcd/vendor -o -path ./origin -o -path ./deps \) -prune -o -name '*.py' -print)
+    pyfiles=$(find . -type d \( -path ./_output -o -path ./vendor -o -path ./assets -o -path ./etcd/vendor -o -path ./origin -o -path ./deps -o -path ./scripts/zstd \) -prune -o -name '*.py' -print)
```

Comment:
Can we add the whole `./scripts`, which is more future-proof?

Reply:
Why would we omit linting scripts? If we think it's not useful then maybe we should remove it altogether.

Reply:
No reason. All scripts should be there. 
It might require a separate PR depending on the errors.
@pacevedom , please open a separate issue on this if you see there are errors unrelated to this PR.

Reply:
I excluded this file because its not part of any release/CI job and its output is not consumed by others, but there are others which are. For example the rebase, or AI model serving, or the release notes scripts. Those should remain under linting because they are used.